raiden/__main__.py

```diff
@@ -5,6 +5,13 @@ def main() -> None:
     import gevent.monkey
 
     gevent.monkey.patch_all()
+
+    import asyncio  # isort:skip # noqa
+    import raiden.network.transport.matrix.rtc.aiogevent as aiogevent  # isort:skip # noqa
+
+    asyncio.set_event_loop_policy(aiogevent.EventLoopPolicy())  # isort:skip # noqa
+    gevent.spawn(asyncio.get_event_loop().run_forever)  # isort:skip # noqa
+
```

Comment:
This code is repeated a lot in the `tools`. Sounds like a good candidate to move to a utility function.